backend/fleet_management/management/commands/populate_database.py

```diff
@@ -19,31 +19,49 @@ class Command(BaseCommand):
     help = "Populates database with fake items."
 
     def handle(self, *args, **options):
-        """Populate database with fake objects."""
+        superuser = User()
```

Comment:
That's Python equivalence of:
```
var superuser = {};
superuser.is_active = true
superuser.is_superuser = true
superuser.is_staff = true
superuser.username = 'hello@codeforpoznan.pl'
superuser.email = 'hello@codeforpoznan.pl'
superuser.country = 'UA'
```
Would prefer to have that in init:
```
superuser = User(
	is_active=True,
	is_superuser=True,
	is_staff=True,
	username='hello@codeforpoznan.pl',
	email='hello@codeforpoznan.pl',
	country='UA',
)
```

Reply:
Lets discuss where this should go and update it accordingly then.

Reply:
I just checked `create_admin` script and that's for me best approach how to create `User` objects:
```
user = User.objects.create_user(
    username=options['username'],
    email=options['username'],
    password=options['password'],
    is_superuser=options['django_admin'],
    is_staff=options['django_admin'],
    is_active=True,
)

```

Reply:
What is the difference?

Reply:
I would say stylistic. I do not know anybody with extensive experience in Python that set attributes after creating an empty object as you did it here.